### ISSUE TITLE
[DropdownMenu]: Keyboard Navigation doesn't work properly

### ISSUE THREAD

**Issue by seongminn** [radix-ui/primitives#3640]
### Current Behavior

According to the W3C documentation, when the Arrow Up key is pressed on a DropdownMenu Trigger, the content should open, and focus should be set on the last item. ([Reference](https://www.w3.org/WAI/tutorials/menus/application-menus/#top-level-menu-items))

However, currently, pressing the Arrow Up key does not open the content.

### Expected behavior

When the Trigger is focused and the Arrow Up key is pressed, the content should open, and focus should move to the last item.

### Reproducible example

https://www.radix-ui.com/primitives/docs/components/dropdown-menu

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |    @radix-ui/react-dropdown-menu     |  v2.1.15       |
| React            | n/a     |         |
| Browser          |      chrome   |         |
| Assistive tech   |         |         |
| Node             | n/a     |         |
| npm/yarn/pnpm    |      pnpm   |         |
| Operating System |        macos |         |
